rosdep/python.yaml

```
@@ -465,6 +465,10 @@ pyqt5-dev-tools:
   openembedded: ['${PYTHON_PN}-pyqt5@meta-qt5']
   rhel: [python3-qt5-devel]
   ubuntu: [pyqt5-dev-tools]
+pyqt6-dev-tools:
+  debian: [pyqt6-dev-tools]
+  fedora: [python3-pyqt6-base]
+  ubuntu: [pyqt6-dev-tools]
```

Right, so this package doesn't exist for Ubuntu Focal and Jammy.

In that case, this needs to be:

```suggestion
  ubuntu:
    '*': [pyqt6-dev-tools]
    focal: null
    jammy: null
```